src/models/todo.ts

```diff
@@ -1,9 +1,9 @@
 interface Todo {
```

Comment:
You should probably separate types, one that comes from API always has id, createdAt and updatedAt. The ones that you create in don't have those.